types/dagre/index.d.ts

```diff
@@ -12,15 +12,27 @@ export namespace graphlib {
         nodes(): string[];
         node(id: any): any;
         setDefaultEdgeLabel(callback: () => void): Graph;
-        setEdge(sourceId: string, targetId: string, options?: { [key: string]: any }): Graph;
+        setDefaultNodeLabel(callback: () => void): Graph;
```

Comment:
From the source it looks like `setDefaultNodeLabel` can take either a [constant value _or_ a function](https://github.com/cpettitt/dagre/blob/master/dist/dagre.js#L3588), and that the function has a return value that is [non-`void`](https://github.com/cpettitt/dagre/blob/master/dist/dagre.js#L3635).